### ISSUE TITLE
cleartext signatures

### ISSUE THREAD

**Issue by raboof** [sbt/sbt-pgp#143]
It would be cool if sbt-pgp had support for cleartext signatures, as described at http://tools.ietf.org/html/rfc4880#section-5.2.4 and http://tools.ietf.org/html/rfc4880#section-7 .

**Comment by djspiewak** [comment]
As far as I know, all tools which sign content for Maven repositories perform binary signing. More importantly, all tools which *verify* said signatures do so using the binary standard. Why is it you're looking for cleartext signing?

**Comment by raboof** [comment]
I'm playing with https://github.com/raboof/sbt-reproducible-builds, and the reproducible builds convention is to share the build metadata and the package hash as a cleartext-signed 'buildinfo' file.

But perhaps it doesn't really make sense to try and reuse sbt-pgp for that, seems tricky.

**Comment by djspiewak** [comment]
@raboof Yeah I definitely wouldn't reuse sbt-pgp there. You could reuse sbt-gpg instead, which is a considerably simpler plugin that gives you more control over the `gpg` invocation, or just call `gpg` directly.
